tests/test_lnchannel.py

```diff
@@ -721,6 +730,59 @@ def test_DesyncHTLCs(self):
         self.assertEqual(500000000000, bob_channel.available_to_spend(LOCAL))
         alice_channel.add_htlc(htlc)
 
+    def test_single_payment(self):
+        alice_channel, bob_channel = create_test_channels(
+            anchor_outputs=self.TEST_ANCHOR_CHANNELS,
+            local_msat=bitcoin.COIN * 40,
+            remote_msat=bitcoin.COIN * 40,
+            local_max_inflight=bitcoin.COIN * 10,
+            remote_max_inflight=bitcoin.COIN * 20)
```

Comment:
This is very confusing. 
If we had proper types, bitcoin.COIN would be defined as sats, not as an integer. 
It what you want is 40 mBTC expressed in msats, it would be less confusing to write it as 4000000000.